src/linebreakdef.h

```diff
@@ -161,7 +161,8 @@ struct LineBreakContext
 };
 
 /* Declarations */
-extern const struct LineBreakProperties lb_prop_default[];
+extern const struct LineBreakProperties lb_prop_supplementary[668];
```

Comment:
Changing *linebreakdef.h* every time the data is updated is probably not a good idea. I think you can move this declaration into something like *linebreakdata.h*.

Reply:
It is also a problem that the number is not currently updated by the Python script.